src/main/crypto/secrets.ts

```diff
@@ -0,0 +1,14 @@
+export class Secrets {
+
+  readonly primary: string
+  readonly secondary: string | void
+
+  constructor (primary: string, secondary?: string) {
```

Comment:
I think this would be slightly nicer:
```
constructor (public readonly primary: string, public readonly secondary?: string)
```
then remove the fields and I don't think the void type is right it should just have the nullable `?`